app/components/UserAttendance/AttendanceModalContent.css

```diff
@@ -1,16 +1,34 @@
-.modal {
-  height: calc(100vh - 100px - 70px);
+@import url('~app/styles/variables.css');
+
+.modalContent {
+  /* 
+   * Kinda hacky solution, but with the current flexibility of modal sizing it's the only way I could think of for now
+   * 
+   * 100vh - 100px          = height of the modal from @lego-bricks/src/components/Modal/Modal.module.css
+   * 2 * var(--spacing-xl)  = the padding of the modal
+   * 2rem                   = the height of the title
+   * var(--spacing-md)      = the margin-bottom of the title
+  */
+  height: calc(
+    (100vh - 100px) - (2 * var(--spacing-xl)) - (2rem) - var(--spacing-md)
+  );
 
   @media (--tall-viewport) {
-    max-height: calc(100vh - 300px - 70px);
+    height: calc(
+      (100vh - 200px) - (2 * var(--spacing-xl)) - (2rem) - var(--spacing-md)
```

Comment:
I guess this could be a variable only in this file. And then use calc to remove another `100px` here